src/java/voldemort/server/VoldemortServer.java

```diff
@@ -386,10 +386,8 @@ protected void startInner() throws VoldemortException {
         boolean goOnline;
         if (getMetadataStore().getServerStateUnlocked() == MetadataStore.VoldemortState.NORMAL_SERVER)
```

Comment:
Rebalance Server should still come as online. This would make the rebalance server to go offline.